Makefile

```
@@ -42,7 +42,12 @@ clean:
 build:
 	@poetry build
 
-publish:
+publish-to-jfrog:
 	@poetry publish --repository jfrog --username $(ARTIFACTORY_USERNAME) --password $(ARTIFACTORY_API_KEY)
 
+publish-to-pypi:
+	@poetry config pypi-token.pypi $(PYPI_TOKEN) && @poetry publish
```

We can skip the `config` command as follows:

```suggestion
	@poetry publish --username __token__ --password $(PYPI_TOKEN)
```